services/venice-controller/src/main/java/com/linkedin/venice/controller/StoreBackupVersionCleanupService.java

```diff
@@ -107,6 +106,12 @@ protected StoreBackupVersionCleanupService(
     this.httpAsyncClient = HttpAsyncClients.custom()
         .setDefaultRequestConfig(RequestConfig.custom().setSocketTimeout(10000).build())
         .setSSLContext(sslFactory.map(SSLFactory::getSSLContext).orElse(null))
+        .setKeepAliveStrategy(new DefaultConnectionKeepAliveStrategy() {
```

Comment:
This is to make client connection alive for 1h? Just curious why do we need this.

Reply:
yeah, just an attempt to keep connection open longer